core/src/display_object/movie_clip.rs

```diff
@@ -2855,7 +2855,7 @@ impl<'gc, 'a> MovieClipData<'gc> {
         context: &mut UpdateContext<'_, 'gc, '_>,
         reader: &mut SwfStream<'a>,
     ) -> DecodeResult {
-        let font = reader.read_define_font_2(2)?;
+        let font = reader.read_define_font_2_or_3(2)?;
```

Comment:
In retrospect, I'd prefer to name this simply `read_define_font_3` to be less wordy and to indicate the max version it handles. Ditto for any other methods in SWF that do `m_or_n` -- name them `n` for simplicity.

Reply:
Ack, for now I excluded the relevant commit. I guess we'd want to make this change widely in a separate PR.